jsonschema-hyperschema.xml

```diff
@@ -279,1272 +367,850 @@
-                </t>
-            </section>
+                <section title="headerSchema" anchor="headerSchema">
+                    <t>
+                        <cref>
+                            As with "targetHints", this keyword is somewhat under-specified
+                            to encourage experimentation and feedback as we try to balance
+                            flexibility and clarity.
+                        </cref>
+                    </t>
+                    <t>
+                        If present, this property is a schema for protocol-specific request
+                        headers or analogous control and meta-data.  The value of this
+                        object MUST be a valid JSON Schema.
+                        The protocol is determined by the "href" URI scheme, although note that
+                        resources are not guaranteed to be accessible over such a protocol.
+                        The schema is advisory only; the target resource's behavior is not
+                        constrained by its presence.
+                    </t>
+                    <t>
+                        The purpose of this keyword is to advertise target resource interaction
+                        features, and indicate to clients what headers and header values are
+                        likely to be useful.  Clients MAY use the schema to validate relevant
+                        headers, but MUST NOT assume that missing headers or values are forbidden
+                        from use.  While schema authors MAY set "additionalProperties" to
+                        false, this is NOT RECOMMENDED and MUST NOT prevent clients or user agents
+                        from supplying additional headers when requests are made.
+                    </t>
+                    <t>
+                        The exact mapping of the JSON data model into the headers is
+                        protocol-dependent.  However, in most cases this schema SHOULD
+                        specify a type of "object", and the property names SHOULD be
+                        lower-cased forms of the control data field names.  See the
+                        <xref target="HTTP">"JSON Hyper-Schema and HTTP"</xref> section
+                        for detailed guidance on using this keyword with HTTP and
+                        analogous protocols.
+                    </t>
+                    <t>
+                        "headerSchema" is applicable to any request method or command that the
+                        protocol supports.  When generating a request, clients SHOULD ignore
+                        schemas for headers that are not relevant to that request.
+                    </t>
+                </section>
 
-            <section title="title">
-                <t>
-                    This property defines a title for the link.
-                    The value MUST be a string.
-                </t>
+                <section t
```

Comment:
I agree, this looks strange. The default should not be to constrain target behavior.

Reply:
Is this the line you meant to attach this comment to?  I don't follow.

Reply:
No... The actual line is https://github.com/json-schema-org/json-schema-spec/pull/427/files#diff-6f6cc4b69f2e1ecfca964501e7a07e2eR922 (the cref after the paragraph saying "Omitting "submissionSchema" or setting the entire schema to "false" prevents any user agent data from being accepted.").

Reply:
@dlax oh, cool, yeah that makes a lot more sense :-)
I'm continuing to lean towards saying that omitting "submissionSchema" is the same as `true` rather than `false`.

"hrefSchema" needs to default to behaving like `false` because of how it interacts with the rest of the target URI resolution process.  I couldn't get it to work with a default behavior of `true` without needing another keyword to indicate whether the link accepts input at all.  Or schema authors would have to explicitly set it to false to opt out of accepting input, which feels odd.  But maybe that's just because I've been staring at it for too long.